Dockerfile

```diff
@@ -23,11 +23,14 @@ ENV HOME /root
 COPY ./paddle/scripts/docker/root/ /root/
 
 RUN apt-get update && \
-    apt-get install -y git python-pip python-dev openssh-server bison && \
-    apt-get install -y wget unzip tar xz-utils bzip2 gzip coreutils && \
-    apt-get install -y curl sed grep graphviz libjpeg-dev zlib1g-dev && \
-    apt-get install -y python-numpy python-matplotlib gcc g++ liblapack-dev liblapacke-dev && \
-    apt-get install -y automake locales clang-format-3.8 swig doxygen && \
+    apt-get install -y \
+    git python-pip python-dev openssh-server bison  \
+    wget unzip tar xz-utils bzip2 gzip coreutils  \
+    curl sed grep graphviz libjpeg-dev zlib1g-dev  \
+    python-numpy python-matplotlib gcc g++ \
+    automake locales clang-format-3.8 swig doxygen cmake  \
+    liblapack-dev liblapacke-dev libboost-dev \
```

Comment:
I understand and agree that we should use CMake and its external projects. I didn't do it in this PR because I want to figure out what is the command line we need to build Majel.